### DISCUSSION TITLE
multi-select with chips?

### DISCUSSION THREAD

**Discussion by metaperl** [lona-web-org/lona#493] [Q&A]
NiceGUI has [a dropdown multi-select](https://nicegui.io/documentation/select#multi_selection) that has "chips" that you can click an "X" on to remove some of the selection items - https://nicegui.io/documentation/select#multi_selection

Is such an element available in Lona?

**Comment by fscherf** [comment]
Lona aims to provide a simple API for basic HTML nodes. The standard library does not contain such complex components with that level of internal logic. We have libraries on top of Lona like [lona-picocss](https://github.com/lona-web-org/lona-picocss) or [lona-bootstrap-5](https://github.com/lona-web-org/lona-bootstrap-5) to implement higher level components like modals, grid systems, progressbars etc.

For this particular case: I don't know of a library that implements these chips, but it shouldn't be too hard I think. I can provide some help if you want to build something like this
